File: 1-2star/10420 List of Conquests/10420.cpp
#include <iostream>
#include <map>
using namespace std;
int main(){
    int n;
    map<string, int> m;
    cin >> n;
    for(int i = 0; i < n; i++){
        string country;
        string name;

        cin >> country;

        getline(cin, name);
        m[country]++; 
    }
    map<string, int>::iterator it;
    for(it=m.begin(); it!=m.end(); it++)
    {
        cout<<it->first<<" "<<it->second<<endl;
    }
    m.clear();
}

File: 1-2star/10420 List of Conquests/10420.py
n = int(input())  # Read the number of lines
country_counts = {}  # Dictionary to store country counts

for _ in range(n):
    line = input().split()  # Split the input line into words
    country = line[0]  # The first word is the country name
    country_counts[country] = country_counts.get(country, 0) + 1

for country in sorted(country_counts):
    print(country, country_counts[country])
